partcad/tests/unit/test_git_clone_retries.py

```
@@ -0,0 +1,127 @@
+        f"""
+          fatal: unable to access '{repo_url}': Received HTTP code 407 from proxy after CONNECT
+          fatal: unable to access '{repo_url}': Proxy authentication required
+          fatal: unable to access '{repo_url}': Proxy error: 407 Proxy Authentication Required
+        """
+    ),
+]
+
+test_git_retry_config = {"max": 5, "patience": 0.1}
+
+@pytest.mark.parametrize("git_error", fake_git_errors)
+def test_project_import_git_clone_retry_failure(git_error: GitCommandError):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_project_import_git_clone_retry_failure`

Add return type annotation: `None`

(ANN201)